crates/uv/src/commands/run.rs

```diff
@@ -39,14 +39,16 @@ use uv_types::{BuildIsolation, EmptyInstalledPackages, HashStrategy, InFlight};
 /// Run a command.
 #[allow(clippy::unnecessary_wraps, clippy::too_many_arguments)]
 pub(crate) async fn run(
-    command: String,
+    target: Option<String>,
     args: Vec<String>,
     mut requirements: Vec<RequirementsSource>,
     isolated: bool,
     no_workspace: bool,
     cache: &Cache,
     printer: Printer,
 ) -> Result<ExitStatus> {
+    let command = target.unwrap_or("python".to_string());
```

Comment:
I guess we could use the direct path to the Python in the virtual environment but that seems overkill to me?